js/goat.js

```diff
@@ -773,6 +775,36 @@ Goat.prototype.resetActionStates = function (goat) {
     this.stunned = false;
 };
 
+// used by EndGame Scene
+Goat.prototype.drawStillStandingAnimation = function (ctx) {
```

Comment:
This should be encapsulated in endGame or better yet: Instead of using actual "Goats", you can instead just draw the image of the goats since you are not animating them anyways. Use the .drawImage() canvas method